kolibri/core/auth/api.py

```diff
@@ -180,6 +183,19 @@ class FacilityUserViewSet(ValuesViewset):
         "is_superuser": lambda x: bool(x.pop("devicepermissions__is_superuser"))
     }
 
+    @list_route(methods=["get"])
+    def users_for_facilities(self, request):
+        facility_ids = dict(request.GET)["member_of"]
```

Comment:
Shouldn't need to coerce `request.GET` to a dict, and this will give a 500 when `member_of` is undefined.